code/modules/client/preference_setup/general/01_basic.dm

```diff
@@ -104,6 +104,15 @@
 	dat += "<b>Gender:</b> <a href='?src=\ref[src];gender=1'><b>[capitalize(lowertext(pref.gender))]</b></a><br>"
 	dat += "<b>Age:</b> <a href='?src=\ref[src];age=1'>[pref.age]</a><br>"
 	dat += "<b>Spawn Point</b>: <a href='?src=\ref[src];spawnpoint=1'>[pref.spawnpoint]</a><br>"
+	if(is_string_in_list(pref.species, ipc_species_by_name, FALSE))
```

Comment:
seems like a pointless global, just check the species here, we have a species list of name -> species